src/App.js

```diff
@@ -1,8 +1,46 @@
 import React from 'react';
 import './App.css';
+import CharacterInfo from './Components/Character/CharacterInfo';
+import ActionsButton from './Components/Actions/ActionsButton';
+import EnergyButton from './Components/Energy/EnergyButton';
+import CoinsButton from './Components/Coins/CoinsButton';
+import SeenButton from './Components/Seen/SeenButton';
+import CraftingButton from './Components/Crafting/CraftingButton';
+import Map from './Components/Map/Map';
+import DisastersContainer from './Components/Disasters/DisastersContainer';
 
-function App() {
-  return <div className="App">Project created</div>;
+class App extends React.Component {
+  render() {
+    return (
+      <div className="App">
+        <div id="mapContainer">
+          <Map></Map>
+        </div>
+
+        <div id="upperLeftButtonsContainer">
+          <EnergyButton></EnergyButton>
```

Comment:
please change `<EnergyButton></EnergyButton>` to `<EnergyButton/>`.
Same effect with cleaner syntax.
same as with the other components down here.

Reply:
Will do

Reply:
Implemented.